tutor/src/screens/assignment-grade/ux.js

```diff
@@ -162,6 +162,14 @@ export default class AssignmentGradingUX {
     return this.planScores.taskPlan.tasking_plans.forPeriod(this.selectedPeriod);
   }
 
+  @computed get getDropQuestionPoints() {
+    const droppedQuestion = this.selectedHeading.dropped;
+    if(droppedQuestion) {
+      return droppedQuestion.drop_method == 'zeroed' ? 0 : this.selectedHeading.points_without_dropping;
+    }
+    return null;
+  }
+
```

Comment:
I slightly dislike use of a `get` prefix since it almost always doesn't add anything to the name. it could be named `droppedQuestionPoints`

Reply:
:+1: